app/models/mixins/kubernetes_provider_mixin.rb

```diff
@@ -43,7 +38,7 @@ def connect(options = {})
     options[:port] ||= self.port
     options[:user] ||= authentication_userid(options[:auth_type])
     options[:pass] ||= authentication_password(options[:auth_type])
-    options[:bearer] ||= authentication_token(options[:auth_type])
+    options[:bearer] ||= authentication_token("bearer")
```

Comment:
@simon3z  Well, if options is passed, so :auth_type should be "bearer", but when it isn't, it calls authentication_token without it, so we won't get the token for connection.
(I can add  - options[:auth_type] == nil ? "bearer" : options[:auth_type], tell me what you think)